packages/components/input/__tests__/input.test.tsx

```diff
@@ -149,7 +149,7 @@ describe("Input", () => {
       />,
     );
 
-    const clearButton = getByRole("button");
+    const clearButton = container.querySelector("button")!;
```

Comment:
It would be preferable to use `getByRole` wherever possible.
ref: https://testing-library.com/docs/queries/about/#priority

Reply:
Thanks for the suggestion!
I've updated the tests to use getByRole("button") as recommended.